src/Apps/Legion/Contexts/AppDataContext/AppDataContext.types.ts

```diff
@@ -1,4 +1,4 @@
-import { IExtensionClient } from '../../EntryPoints/LegionApp/ShellTypes';
```

Comment:
What would you think about having another "App" called "Core" or "Shared" and putting stuff like this in there?